mdn/developer-portal
# Add toggle to show/hide home page header

This changeset adds support for a home page header toggle. This allows the author to toggle on/off the hero section as desired.

## Key changes

- Add "show_header" field to home model.
- Add conditional to home template to prevent hero rendering if "show_header" is checked.

## How to test

- Toggle on/off the "show_header" field to ensure the hero is displayed/hidden as expected.
